protocol/development-workflow/README.md

```diff
@@ -1,171 +1,75 @@
 # Development Workflow
 
-- [Setting Up the Release Branch](#setting-up-the-release-branch)
-- [Developing for the Release](#developing-for-the-release)
-- [Conducting QA](#conducting-qa)
-  + [Rejecting](#rejecting)
-  + [Accepting and Deploying](#accepting-and-deploying)
-- [Small, discretionary releases](#small-discretionary-releases)
+People: Developer, Code Reviewer, QA-er
 
-## Setting Up the Release Branch
+Tools: Github (GH), Pivotal Tracker (PT), Aptible Staging Servers
 
-As a team, determine the scope of a release. The release can contain one
-Pivotal Tracker story, or it can combine multiple stories that cannot be
-deployed independently of one another. A release will be staged, accepted,
-integrated, and deployed as a single unit.
+## Developer
+1. Create a `release/` branch off the latest version of `master` and push to GH
 
-Create a release branch off of `master`.
+    a. `git pull` for the latest state of `master`
+    
+    b. *Start* the PT story<sup>[0](#fn0)</sup>
```

Comment:
An absolute non-blocker, but everything just reads easier for me (rich HTML wise) if there was a space between the word and its superscript footnote: test <sup>0</sup>